src/utils/styling.js

```diff
@@ -1,13 +1,13 @@
 import colorUtils from './color-utils';
 
-export function getColor(reference, palette, defaultColor) {
+export function getColor(reference, Theme, defaultColor) {
   let color;
   switch (reference.colorType) {
     case 'byExpression':
       color = colorUtils.resolveExpression(reference.colorExpression);
       break;
     case 'colorPicker':
-      color = colorUtils.resolveColor(reference.color, palette);
```

Comment:
I think this is the only place we use this function. We could remove it from utils then

Reply:
ah true, I think Miralem removed it from the action button.